docs/upgrade/upgrade-7.17-8.x.asciidoc

```diff
@@ -0,0 +1,155 @@
+
+* Review release notes, deprecations, and breaking changes for {elastic-sec}, {es}, {kib}, and, if applicable, {fleet}, {agent}, {beats}, and {ls}. Identify any issues that might affect your deployment. Work with your Support Engineer on any questions you may have. Start with breaking changes for solution and platform components you use, such as {es} and {kib}. 
+
+* Schedule a system maintenance window within your organization.
+
+[float]
+=== Pre-upgrade steps
+
+To prepare for the upgrade process, follow these steps before you start:
+
+. Do a software version inventory across your entire Elastic platform, including {es}, {kib}, {agent}, {beats}, and {ls}. If you're managing your deployments using {ece-ref}/ece-upgrade.html[{ece}] (ECE) or {eck-ref}/k8s-upgrading-eck.html[{eck}] (ECK), you may need to upgrade the orchestrator first.
+
+. If you're running any of these at versions earlier than 7.17, you must first upgrade them all to the latest 7.17.x patch release before upgrading to 8.x. This enables you to use the {kib} Upgrade Assistant to upgrade to 8.x. 
+
+. Run the {kib-ref}/upgrade-assistant.html[Upgrade Assistant]. Take note of any critical error messages, then work with your Support rep to resolve them.
++
+.Requirements
+[sidebar]
+--
+To run the Upgrade Assistant, you must have `superuser` cluster admin permissions.
+--
+
+. If you're not using {ref}/snapshots-take-snapshot.html#automate-snapshots-slm[{slm} (SLM)], you must set up, configure, and run at least one {ref}/snapshot-restore.html[snapshot]. Snapshots are {ref}/snapshot-restore.html#how-snapshots-work[incremental] – depending on the cluster size and the input/output rate, the initial snapshot may take several hours to complete. If you are using SLM, {ref}/snapshots-take-snapshot.html#check-slm-history[check the SLM history] to ensure that snapshots are completing successfully.
+
+[float]
+=== Perform 8.x upgrade on Non-Prod (Optional) or Prod Deployment
+
+NOTE: We strongly recommend you perform the following steps on a non-prod test deployment first to address any potential issues before upgrading on a prod deployment. 
+
+. Back up all of your data to a {ref}/snapshots-take-snapshot.html[snapshot] – which is a backup of an index taken from a running cluster. Rollbacks require a snapshot and cause data loss without one.
+
+. We recommend you <<rules-api-export, export all your custom detection rules>> in the event there are issues with the detection engine after the upgrade.
+
+. Upgrade {es}. Refer to {stack-ref}/upgrading-elasticsearch.html[these upgrade instructions]. 
+** If you're using {ecloud}, we recommend upgrades with no downtime. Refer to {cloud}/ec-upgrade-deployment.html[these instructions].  
```

Comment:
```suggestion
** If you're using {ecloud}, refer to {cloud}/ec-upgrade-deployment.html[these instructions].  
```
Ignore this suggestion if the part I deleted is important, but this would be more aligned with the format of the following bullets.

Reply:
Sure - We can eliminate this I think. Thanks.